rmf_schedule_visualizer/src/VisualizerData.hpp

```diff
@@ -20,6 +20,8 @@
 #define RMF_SCHEDULE_VISUALIZER__SRC__VISUALIZERDATA_HPP
 
 #include <rmf_traffic_ros2/schedule/MirrorManager.hpp>
+#include <rmf_traffic_ros2/schedule/Negotiation.hpp>
+#include <rmf_traffic_ros2/schedule/Writer.hpp>
```

Comment:
This won't be needed once `_writer` is removed.

Reply:
removed 3b1c580